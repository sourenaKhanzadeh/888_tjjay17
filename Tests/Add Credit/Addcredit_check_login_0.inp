Enter user type: AA
Login 
Admin
logout

Or 

enter user type: FS,BS,SS
login 
Full standard, Buyer Standard, or Seller Standard
logout
