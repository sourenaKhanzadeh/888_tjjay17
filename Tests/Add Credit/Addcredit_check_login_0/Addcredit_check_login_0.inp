Enter user type: AA,FS,BS,SS
Login 
Admin/ User Standard
Logout


