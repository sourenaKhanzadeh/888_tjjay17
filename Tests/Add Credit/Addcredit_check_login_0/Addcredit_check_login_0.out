Welcome to the add credit page 
Admin, or user standard logged in
Session terminated
